packages/main/src/plugin/index.ts

```diff
@@ -2530,7 +2531,7 @@ export class PluginSystem {
 
     this.ipcHandle(
       'kubernetes-client:registerGetCurrentContextResources',
-      async (_listener, resourceName: ResourceName): Promise<KubernetesObject[]> => {
+      async (_listener, resourceName: ResourceName): Promise<KubernetesObject[] | CoreV1Event[]> => {
```

Comment:
Maybe have a dedicated ipc as events are really not the same as KubernetesObject

Reply:
I realized that CoreV1Event is actually a superset of KubernetesObject, this is finally not necessary